src/model/AssetsCompiler.ts

```diff
@@ -107,10 +108,25 @@ export class AssetsCompiler {
         private readonly runConfig: IAssetsCompilerConfiguration,
         private readonly repositoryDir: string
     ) {
+        if (!AssetsCompiler.shouldUseAscLocal()) {
+            console.warn(
+                cwarn`@cplace/asc-local should only be used starting from the cplace release 23.2!`
+            );
+        }
         this.repositoryName = path.basename(repositoryDir);
         this.projects = this.setupProjects();
     }
 
+    private static shouldUseAscLocal(): boolean {
+        const version = CplaceVersion.get();
+        const isMajorHigherThen23 = version.major > 23;
+        const isMinorHigherOrEqual2 =
+            version.major === 23 && version.minor >= 2;
+        const is23_1Snapshot =
+            version.major === 23 && version.minor === 1 && version.snapshot;
+        return isMajorHigherThen23 || isMinorHigherOrEqual2 || is23_1Snapshot;
+    }
```

Comment:
Same stuff as [here](https://github.com/collaborationFactory/cplace-asc/pull/98#discussion_r1104641862) and [here](https://github.com/collaborationFactory/cplace-asc/pull/98#discussion_r1104637671) 👀